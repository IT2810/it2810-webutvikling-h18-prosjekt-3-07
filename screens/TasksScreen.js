
import React from 'react';
import {Text, View, StyleSheet} from 'react-native';
import TodoList from '../components/TodoList';

class TasksScreen extends React.Component {
  constructor(props) {
    super(props);
    }
    static navigationOptions= {
      title: 'Tasks',
    };
    render() {
    
      return (
        <View style={styles.container}>
         <TodoList></TodoList>
        </View>

      );
    }
  }
  
  const styles = StyleSheet.create({
    container: {
      flex: 1,
      alignItems: "center",
      justifyContent: "center",
      backgroundColor: '#fff',
    }
  });

  export default TasksScreen;