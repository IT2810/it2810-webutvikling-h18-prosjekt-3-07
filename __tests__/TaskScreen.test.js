import React from "react";
import ShallowRenderer from "react-test-renderer/shallow";
import TasksScreen from "../screens/TasksScreen";

it("matches snapshot", () => {
  const renderer = new ShallowRenderer();
  const result = renderer.render(<TasksScreen />);
  expect(result).toMatchSnapshot();
});
