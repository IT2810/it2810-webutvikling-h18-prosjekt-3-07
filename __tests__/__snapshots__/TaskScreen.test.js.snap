// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<Component
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <TodoList />
</Component>
`;
